modules/core/server/views/partials/footer.server.view.html

```diff
@@ -66,7 +66,7 @@
           </a>
           {% else %}
           <a
-            href="https://github.com/Trustroots/trustroots"
+            href="https://github.com/OpenHospitalityNetwork/fedi-trustroots"
```

Comment:
Maybe nice to put this too in the constants.js file?

Reply:
definitely a good idea. I think the defaults.js is better but still working on figuring that out, see the wiki...